From d4a180e5b67c3ca9b8559d4f926f22b6c6705082 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date: Mon, 10 Jun 2019 16:32:51 +0100
Subject: [PATCH] drm/vc4: Remove 340MHz clock limit from FKMS now
 scrambling issues resolved

Firmware TMDS scrambling is now being correctly configured, so
we can use it.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>
---
 drivers/gpu/drm/vc4/vc4_firmware_kms.c | 6 ------
 1 file changed, 6 deletions(-)

--- a/drivers/gpu/drm/vc4/vc4_firmware_kms.c
+++ b/drivers/gpu/drm/vc4/vc4_firmware_kms.c
@@ -830,12 +830,6 @@ vc4_crtc_mode_valid(struct drm_crtc *crt
 		break;
 	}
 
-	/* Limit the pixel clock until we can get dynamic HDMI 2.0 scrambling
-	 * working.
-	 */
-	if (mode->clock > 340000)
-		return MODE_CLOCK_HIGH;
-
 	return MODE_OK;
 }
 
